src/definitions/modules/modal.less

```diff
@@ -607,13 +606,13 @@
   border-top: @invertedActionBorder;
 }
 
-.ui.fullscreen.inverted.modal > .close {
-  color: @invertedInnerCloseColor;
```

Comment:
This removes all usages of `@invertedInnerCloseColor`. Is this correct?

Reply:
I didn't unterstand why the css rule was removed, but I just saw that there is a rule overriding this anyway:
https://github.com/fomantic/Fomantic-UI/blob/2c3bc578defd5fe92e6e47fbdb795636a935e105/src/definitions/modules/modal.less#L620-L622

Yes, please remove it from modal.variables
@hammy2899 @prudho Because the variable was essentially not used before, this isn't a breaking change, right?

Reply:
For me, now, since removing it will not have any effect anywhere.